markdown/articles/2021/08/22/Non-blue-fair-decks.md

```diff
@@ -0,0 +1,429 @@
+
+#### Post Doomsday 
+
+## Bant Miracles
+
+### Gameplan
+
+### Cards to keep in mind
+
+#### Maindeck
+
+<row variant="hand"> {{!Endurance}} {{!Force of Will}} {{!Force of Negation}} {{!Dress Down}} {{!Teferi, Time Raveler}}</row>
+
+#### Sideboard
+
+<row variant="hand"> {{!Hullbreacher}} {{!Ethersworn Canonist}} {{!Surgical Extraction}} {{!Force of Negation}} {{!Meddling Mage}} {{!Torpor Orb}} {{!Back to Basics}}  {{!Flusterstorm}}</row>
+
+### Gameplay
+
+#### Pre Doomsday
+
+#### Post Doomsday 
+
+## Death and Taxes
+
+### Gameplan
+
+Slam early and build your pile around mana and card draw denial.
+
+### Cards to keep in mind
+
+#### Maindeck
+
+<row variant="hand"> {{!Kaldra Compleat}} {{!Spirit of the Labyrinth}} {{!Rishadan Port}} {{!Wasteland}} {{!Solitude}}</row>
+
+#### Sideboard
+
+<row variant="hand"> {{!Deafening Silence}} {{!Mindbreak Trap}} {{!Surgical Extraction}}</row>
+
+### Gameplay
+
+#### Pre Doomsday
+
+Against DnT resolving an early {{Doomsday}} is the safest way to win. Mulliganing for a Turn 1 or Turn 2 Doomsday is a good starting point.  
+Longer games are their only reliable way to win. They deny our mana and hinder our cantrips.  
+Game 1 they have no way to interact with your Doomsday on the stack, so you can just slam as early as possible.  
+In post board games play around {{Mindbreak Trap}} when it's reasonable and keep countermagic ready to fight {{Deafening Silence}}.  
+
+#### Post Doomsday 
+
+it's very important to keep in mind, that your mana is your biggest weakness and that you can not rely on cyclers as tool to work through your pile.  
+There are many possible piles, but one of the easiest, most tested and pretty {{Surgical Extraction}} proof pile is Fetchland, Fetchland, Thassa's Oracle, Land, Land.  
+It gives you the necessary mana, thins the pile and goes down to 0 cards to play around {{Solitude}}. All without cyclers or cantrips that get stuck in your hand against a {{Spirit of the Labyrinth}}.  
+It's also very customizable. For instance if they already build up some pressure, add a {{Massacre}} or a removal spell for {{Stoneforge Mystic}} to stay safe from {{Kaldra Compleat}}.  
+When we are already talking about Kaldra Compleat, remember it has haste and represents 5 damage for 2 Mana out of a Stoneforge Mystic.  
+
+## Maverick
+
+### Gameplan
+
+An early Doomsdays gives you enough time to build safe piles.
+
+### Cards to keep in mind
+
+#### Maindeck
+
+<row variant="hand"> {{!Gaddock Teeg}} {{!Collector Ouphe}} {{!Endurance}} {{!Wasteland}} {{!Solitude}} {{!Dark Depths}}</row>
+
+#### Sideboard
+
+<row variant="hand"> {{!Deafening Silence}} {{!Mindbreak Trap}} {{!Surgical Extraction}} {{!Choke}} {{!Thoughtseize}}</row>
+
+### Gameplay
+
+#### Pre Doomsday
+
+The matchup against Maverick plays pretty similar to the DnT mactchup, but with a small but important difference, the "I-Win" Button {{Dark Depths}}. Luckily the Dark Depths lines are all very telegraphed and easy to spot.  
```

Comment:
Are we lumping Mav and GW depths together? I see them as quite different myself.

Reply:
GW Depths has it's own section, but many current Maverick lists i found online had Depths as 2nd plan

Reply:
I think maybe keep it out for now?

Reply:
ok